### DISCUSSION TITLE
Welcome to Trading-View-Indicators Discussions!

### DISCUSSION THREAD

**Discussion by ChintzRuparel** [TWODS-CAPITAL/Trading-View-Indicators#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 123solanki** [comment]
I need Lux algo indicators

**Comment by ChintzRuparel** [comment]
Hey @123solanki, 

As we all know Lux Algorithmic Indicators are specific to Trading View Pro Users only and putting them on Open Source Collaboration Apps like GitHub would be an infringement under Trading View's Terms and Conditions. If Any or sort of help is required please feel free to contact me @ChintzRuparel 

I hope I was of some help. 

Thanking You, 
Chintz Ruparel
